docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+"""
+Train Test Validation Scenario on Lending Club Data - Quickstart
```

Comment:
```suggestion
Train-Test Validation Suite on Lending Club Data - Quickstart
```
Now it will match the data integrity one.
Also, isn't it a bit complicated for a title? And weird for the "quickstart" to be in the end. What about "Quickstart - Train Test Validation Suite" and not mention data source?

Reply:
Shortened according to proposal. left data source in brackets for context
(maybe it will make users "relate" to a specific scenario or demonstration. once we have good use cases can drop it here)